index.html

```diff
@@ -19,37 +19,39 @@
     <h1 class="text-center mt-2 text-xl text-gray-200 font-semibold">
       <a
         href="https://www.instagram.com/frederiquelamazone/"
-        class="rounded-lg bg hover:text-blue-800 hover:bg-white"
+        class="rounded-lg bg hover:text-blue-800 text-shadow"
```

Comment:
Text is blue on hover => very bad contrast.
Text-shadow in red is not readable!

![image](https://user-images.githubusercontent.com/1632391/61037122-be1fff00-a3ca-11e9-879e-c396e5c87ed7.png)

![image](https://user-images.githubusercontent.com/1632391/61037111-b9f3e180-a3ca-11e9-83c9-5c6fd28f0d1b.png)

Reply:
I don't see your commit